app/views/courses/salary-details.njk

```diff
@@ -29,7 +29,7 @@
       <p class="govuk-body">You should:</p>
       <ul class="govuk-list govuk-list--bullet">
         <li>
-          give an indication of the salary – if you don’t have the figure, say how it will be calculated (eg using the unqualified teachers’ pay scales)
+          tell candidates the salary or how it will be calculated - for example, using the unqualified teachers' national pay scales
```

Comment:
It might be good to link to the pay scale here.

Reply:
For example:

```
tell candidates the salary or how it will be calculated - for example, using the <a class="govuk-link" href="LINK TO PAY SCALE" target="_blank">unqualified teachers' national pay scales (opens in a new tab)</a>
```